### PR TITLE
Update Selwyn PPP

### LINKED ISSUE #239: Add PPP [Selwyn]

### PR DESCRIPTION
**Pull request by SelwynAng**
Closes #239 

- Updated existing PPP for Selwyn

### CODE REVIEW

File: docs/team/selwynang.md
Comment (wr1159) [inline docs/team/selwynang.md]: Do you want to include more / another pr review comment since the review was on your own pr and might not be very representative?

**Review by wr1159 — COMMENTED** [review]
LGTM, just one small comment. Should we wait until all the PRs are done before merging this?

File: docs/team/selwynang.md
Comment (SelwynAng) [reply docs/team/selwynang.md]: Alright sure!

**Comment by SelwynAng** [comment]
> LGTM, just one small comment. Should we wait until all the PRs are done before merging this?

Yes, we will merge all PPPs together after all bugs have been fixed. We can do it while finalising our UG and DG.

File: docs/team/selwynang.md
Comment (andrechuakj) [inline docs/team/selwynang.md:65]:
```suggestion
    * UC04 - List all events
```

**Review by dylkaw — APPROVED** [review]
LGTM! We can always make more edits later since this is a draft.

**Review by andrechuakj — COMMENTED** [review]
Just a nit, LGTM

**Pull request merged**
